spaces/domains.json

```
@@ -526,6 +526,8 @@
   "vote.huckleberry.finance": "huckleberrydex.eth",
   "vote.hectordao.com": "hectordao.eth",
   "gov.tethys.finance": "tethysswap.eth",
+  "vote.darkmatterdefi.com": "dmdarkmatter.eth",
+  "gov.alluo.io": "alluo.eth"
```

You will miss a comma at the end of he 
```suggestion
  "gov.alluo.io": "alluo.eth",
```

---

oh i didn't see that updated